repo/packages/S/sysdigcloud/0/marathon.json.mustache

```diff
@@ -0,0 +1,65 @@
+{
+    "id": "{{sysdigcloud.app_id}}",
+    "cpus": {{sysdigcloud.cpus}},
+    "constraints": [["hostname", "UNIQUE"]],
+    "mem": {{sysdigcloud.mem}},
+    "instances": {{sysdigcloud.instances}},
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "{{resource.assets.container.docker.agent}}",
+            "network": "HOST",
+            "privileged": true,
+            "parameters": [
+                {
+                    "key": "name", "value": "sysdig-agent"
+                },
+                {
+                    "key": "env", "value": "ACCESS_KEY={{sysdigcloud.access_key}}"
```

Comment:
Can you pass env via. `env` instead of docker params? It'll be a little more flexible.